integration/spark/src/main/scala/org/apache/spark/sql/CarbonSqlParser.scala

```diff
@@ -864,8 +864,8 @@ class CarbonSqlParser()
    /**
     * detects whether double or decimal column is part of dictionary_exclude
     */
-  def isDoubleDecimalColDictionaryExclude(columnDataType: String): Boolean = {
-    val dataTypes = Array("double", "decimal")
+  def isNumberInColDictionaryExclude(columnDataType: String): Boolean = {
+     val dataTypes = Array("double", "decimal", "int")
```

Comment:
Add a reverse check that only string or (varchar-check) is only supported. As other datatypes like bigint etc will have problem in above check